vue-frontend/src/components/client-thankyou/BenefitSection.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <section class="tw-relative tw-mt-[20px] tw-mb-[150px]">
+    <h1
+      class="tw-container tw-font-inter-bold tw-text-black-core/[0.87] tw-text-center tw-text-[1.875rem] md:tw-text-[2.813rem] lg:tw-text-[3.438rem] tw-leading-[2.438rem] md:tw-leading-[4.063rem] lg:tw-leading-[5.156rem]"
+    >
+      Here are some of the things you will get by working with us that make us
+      special according to our past customers.
+    </h1>
+    <div class="tw-absolute lg:tw-hidden">
+      <img
+        :src="benefitbgImages[0]"
+        :srcset="`${benefitbgImages[0]} 400w`"
+        loading="lazy"
+        class="tw-w-full tw-h-full tw-object-cover"
```

Comment:
![Screenshot from 2023-05-22 14-46-04](https://github.com/canopas/canopas-website/assets/69897605/9b6b6194-1e7c-40db-b252-da6702f635e9)

There is space below footer due to background image of benefits section